plugin.rb

```diff
@@ -73,7 +75,19 @@ class Engine < ::Rails::Engine
 
   add_admin_route 'discourse_automation.title', 'discourse-automation'
 
+  add_user_api_key_scope(
```

Comment:
This whole user API key thing really confused me (mostly because the process for generating one in core was nightmarish and I couldn't get it to work without debugging hacks), but in this case the new route has an Admin constraint, so unless the user is also admin this won't work.

Should we just be using `add_api_key_scope` instead so the admin can create these?

Reply:
Yes I got confused too, I think we had the same experience.

I think I will go with an API key instead yes :+1:

Reply:
Great! I could not for the life of me to get the public key to work, and yeah I expected that list of admin API scopes in the UI to show the one for automation too.